content/en/docs/installation/_index.md

```diff
@@ -6,6 +6,6 @@ type: "docs"
 ---
 
```

Comment:
suggestion: I think the new "scenarios" page could actually just be placed here, on the index page for the `Installation` section. I'm not sure what we gain by having "scenarios" on a different page.